.github/workflows/compliance-ncs.yml

```diff
@@ -0,0 +1,62 @@
+name: Compliance NCS
+
+on: pull_request
+
+jobs:
+  compliance_job:
+    runs-on: ubuntu-latest
+    name: Run compliance checks on patch series (PR)
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+        with:
+          path: ncs/sdk-zephyr
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: cache-pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-doc-pip
+
+      - name: Run Compliance Tests
+        env:
+          BASE_REF: ${{ github.base_ref }}
+        working-directory: ncs
+        run: |
+          pip3 install -U pip
+          pip3 install -U setuptools
+          export PATH="$HOME/.local/bin:$PATH"
+          pip3 install -U west
+          west init -m https://github.com/nrfconnect/sdk-nrf
+          west update zephyr nrfxlib
+          pip3 install -U gitlint
+          pip3 install -r zephyr/scripts/requirements-compliance.txt
+          cd sdk-zephyr
+          ZEPHYR_BASE=$(pwd) ../zephyr/scripts/ci/check_compliance.py -m Codeowners -m Devicetree -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m Kconfig -c origin/${BASE_REF}.. || true
```

Comment:
no need for the `$(pwd) ../zephyr/` since you are already in `zephyr/`

Reply:
No, I am in `sdk-zephyr`. The reason for both is that `zephyr` is fetched by `west` and `sdk-zephyr` is the clone of the current repo. If I don't set the variable on the run, it will try to execute in `zephyr` which is a whatever `west` brought in and not what the contents of the current PR are.